package.json

```diff
@@ -49,8 +50,18 @@
     "tape": "~4.0.0"
   },
   "scripts": {
-    "test": "tape test/*-test.js | faucet"
+    "install": "node-pre-gyp install --fallback-to-build",
+    "test": "tape test/*-test.js | faucet",
+    "prebuild": "node-pre-gyp rebuild && node-pre-gyp package"
   },
   "license": "MIT",
-  "gypfile": true
+  "gypfile": true,
+  "bundledDependencies": [
```

Comment:
node-pre-gyp needs to be a bundledDep (sadly) because otherwise it might not be usable (fully installed) by the time npm calls out to it. See https://github.com/mapbox/node-pre-gyp/issues/162 for ideas about avoiding this (and being able to safely have node-pre-gyp dynamically installed and still available in time)